python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -245,12 +245,12 @@ async def check_ccw_following_error(self) -> None:
         from the low-level controller.
         """
         try:
-            rot_data = self.tel_rotation.data
-            rot_tai = rot_data.timestamp
-
             # Get camera cable wrap telemetry and check its age.
-            ccw_data = self.mtmount_remote.tel_cameraCableWrap.get()
-            if ccw_data is None:
+            try:
+                ccw_data = await self.mtmount_remote.tel_cameraCableWrap.next(
+                    flush=True, timeout=MAX_CCW_TELEMETRY_AGE
+                )
+            except asyncio.TimeoutError:
```

Comment:
What will happen in the edge case that the `ccw_data` is `None` (no data), and you call the `flush=True`? Will it throw out an error different from `asyncio.TimeoutError` ?